app/views/pages/about.html.erb

```diff
@@ -65,56 +78,51 @@
       </a>
     </p>
     <%- end %>
-
+    <p>
+      Libraries.io tracks releases from <strong><%= number_with_delimiter(Project.total) %></strong> projects on <strong><%= PackageManager::Base.platforms.length %></strong> package managers in one place. Subscribe to a project and receive notifications of new versions or tags for platform like Bower and Go that don't store release information centrally. 
+    </p>
+    <h2>Supported Package Managers</h2>
     <hr>
+    <p>
+      Libraries.io supports the following package managers. For more detail check out <a href='<%= url_for(controller: 'pages', action: 'pmmatrix',) %>'>our compatibility matrix</a>.
```

Comment:
name that you have a named route (`as: : pmmatrix`) you can use `pmmatrix_path` in the views, although that should be renamed to `compatibility_path` as well